.github/workflows/ci.yml

```diff
@@ -56,3 +66,11 @@ jobs:
 
     - name: coverage
       run: pip install codecov && codecov --token=3e56598d-bfe8-4741-a973-f4b70bd2c280
+
+    - name: SonarCloud Scan
+      # Only executed for one of the tested python version
+      if: ${{ always() && matrix['python-version'] == 3.6 }}
```

Comment:
Pourquoi le `always() && ` ?

Reply:
Héhéh bonne question, c'est une subtilité de GitHub Actions
De base, une step n'est executée que lorsque la précédente est réussie.
Hors dans ce cas, même si les tests failent, j'aimerai que le scaner sonar s'execute et uploade une analyse.
Pour cela, il faut utiliser `always()` https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#always